### DISCUSSION TITLE
9AC63A62-869A-89C9-1E27-86062DEEE038

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#180] [General]
# 1908 Tunguska meteor

The Tunguska event was a 12 megaton explosion that occurred over the vast sparsely-populated taiga of Siberia, Russia near the Tunguska River in 1908-Jun-30. Although it did not hit the surface, it was the largest known impact event in recorded history, theorized to have been caused by a small asteroid or cometary fragment about 50-60 m in diameter
NOTE: To view this event, adjust your current Date/Time on or before 1908-Jun-30, 00:12 UTC

https://celestia.mobi/addon?item=9AC63A62-869A-89C9-1E27-86062DEEE038&lang=es

**Comment by Martinariel35** [comment]
Lo necesito
